subt_ign/src/GameLogicPlugin.cc

```diff
@@ -755,6 +823,7 @@ void GameLogicPluginPrivate::OnRockFallDeployRemainingEvent(
           << "  model: " << name << std::endl;
 
         this->LogEvent(stream.str());
+        this->PublishRobotEvent("max_rock_falls", name);
```

Comment:
Rock falls should be published on `RegionEvent`. Also, can the `max_rock_falls` event be utilized to _avoid_ publishing a `rock_fall` event if the rocks did not actually fall? i.e., instead of publishing `rock_fall` and `max_rock_falls`, we would not publish anything for the detection event that did not result in any rocks being dropped.

Reply:
Fixed in a97d4bdfc38aa5d1d9d4d160c21f349c51e1c7f6 and 7aa282b813ad4a3637fee9302cd3b2cfe299b453